<div class="article fulltext-view" itemprop="articleBody"><span class="highwire-journal-article-marker-start"></span><h1 id="article-title-1" itemprop="headline">Further Insights into the Roles of GTP and the C Terminus of the Hepatitis C Virus Polymerase in the Initiation of RNA Synthesis<a id="xref-fn-10-1" class="xref-fn" href="#fn-10">*</a></h1>
   <div class="section abstract" id="abstract-1" itemprop="description">
      <h2>Abstract</h2>
      <p id="p-6">The hepatitis C virus (HCV) NS5b protein is an RNA-dependent RNA polymerase essential for replication of the viral RNA genome.
         <em>In vitro</em> and presumably <em>in vivo</em>, NS5b initiates RNA synthesis by a <em>de novo</em> mechanism. Different structural elements of NS5b have been reported to participate in RNA synthesis, especially a so-called
         “β-flap” and a C-terminal segment (designated “linker”) that connects the catalytic core of NS5b to a transmembrane anchor.
         High concentrations of GTP have also been shown to stimulate <em>de novo</em> RNA synthesis by HCV NS5b. Here we describe a combined structural and functional analysis of genotype 1 HCV-NS5b of strains
         H77 (subtype 1a), for which no structure has been previously reported, and J4 (subtype 1b). Our results highlight the linker
         as directly involved in lifting the first boundary to processive RNA synthesis, the formation of the first dinucleotide primer.
         The transition from this first dinucleotide primer state to processive RNA synthesis requires removal of the linker and of
         the β-flap with which it is shown to strongly interact in crystal structures of HCV NS5b. We find that GTP specifically stimulates
         this transition irrespective of its incorporation in neosynthesized RNA.
      </p>
   </div>
   <ul class="kwd-group">
      <li class="kwd"><span><a class="kwd-search" href="http://www.jbc.org/search?fulltext=Crystal+Structure&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">Crystal Structure</a></span></li>
      <li class="kwd"><span><a class="kwd-search" href="http://www.jbc.org/search?fulltext=Hepatitis+Virus&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">Hepatitis Virus</a></span></li>
      <li class="kwd"><span><a class="kwd-search" href="http://www.jbc.org/search?fulltext=Positive-strand+RNA+Viruses&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">Positive-strand RNA Viruses</a></span></li>
      <li class="kwd"><span><a class="kwd-search" href="http://www.jbc.org/search?fulltext=RNA+Polymerase&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">RNA Polymerase</a></span></li>
      <li class="kwd"><span><a class="kwd-search" href="http://www.jbc.org/search?fulltext=RNA+Synthesis&amp;sortspec=date&amp;submit=Submit&amp;andorexactfulltext=phrase">RNA Synthesis</a></span></li>
   </ul>
   <div class="section intro" id="sec-1">

      <h2>Introduction</h2>

      <p id="p-7">Hepatitis C virus (HCV)<a id="xref-fn-13-1" class="xref-fn" href="#fn-13"><sup>7</sup></a> is a member of the Flaviviridae family that induces severe liver disease in humans (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). The viral genome is a single-stranded RNA of positive polarity containing a single open reading frame (ORF) flanked by
         two untranslated regions (UTRs), the 5′-UTR and 3′-UTR. The single ORF is translated into a large (∼3000 residues) polyprotein
         that is processed into some 10 mature proteins. Thus, the RNA-dependent RNA polymerase (RdRp) NS5b is cleaved from the C terminus
         of the polyprotein. <em>In vivo</em> the 591-residue NS5b is the central player in the synthesis of new genomic RNAs, in association with other viral and cellular
         proteins. This viral replication complex is associated with membranes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>) with the highly hydrophobic C-terminal 21 residues of NS5b forming a transmembrane helix (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). <em>In vitro</em>, NS5b has been shown to be capable of template-directed RNA synthesis on its own, requiring only divalent metals (magnesium
         or manganese) as cofactors. Indeed, NS5b can catalyze both <em>de novo</em> synthesis from a single-stranded template (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>) and primer extension from the subsequent RNA duplex or from a preannealed template/primer duplex. The NS5b C-terminal transmembrane
         helix is dispensable for these activities, and C-terminal deletions of 21 residues (NS5b_Δ21) or more (NS5b_Δ47 to NS5b_Δ60)
         have been used in most activity and all crystallographic studies. The latter (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a><span class="xref-sep">,</span><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">–</a><span class="xref-sep">,</span><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>) has shown that the catalytic core of NS5b comprises residues 1–530 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a><em>E</em>). They have also brought a puzzle to light; the 40-residue stretch (termed “linker” throughout this manuscript) between the
         catalytic core (fingers, palm, and thumb, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>) and the C-terminal membrane anchor occludes the catalytic cleft (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>) in the crystal structures in which it is present (<em>i.e.</em> _Δ21 forms). The only reported exception is the case of the consensus subtype 2a NS5b_Δ21 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>), where two conformations of the same construct were crystallized, one with the linker in its usual, occluding position and
         one with the linker disordered. The arrangement of the linker in the active site of NS5b_Δ21 can be disrupted by mutations
         of critical linker residues interacting with the β-flap (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a><em>B</em>) of the thumb domain (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). The β-flap itself was proposed to participate in initiation of RNA synthesis by selective (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>) and precise (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>) positioning of a single-stranded RNA template along the finger RNA binding cleft, a prerequisite for proper <em>de novo</em> initiation.
      </p>

      <div id="F1" class="fig pos-float odd">
         <div class="fig-inline">
            <a href="32906/F1.expansion.html" class="fig-inline-link"><img alt="FIGURE 1." src="http://www.jbc.org/content/285/43/32906/F1.small.gif"></a>
         </div>
         <div class="fig-caption"><span class="fig-label">FIGURE 1.</span>
            <p id="p-8" class="first-child"><strong>Structures of H77_NS5b in complex with GTP and of J4_NS5b_S556K.</strong> NS5b is in ribbon representation and colored according to domains; fingers are in <em>red</em>, palm is in <em>yellow</em>, thumb is in <em>blue</em>, and linker is in <em>light brown. A–C</em>, shown is H77_NS5b in complex with GTP. GTP molecules are in stick representation and are colored by element; Carbon, <em>green</em>, nitrogen, <em>blue</em>; oxygen, <em>red</em>; phosphorus, <em>orange. A</em> and <em>B</em>, two 90° rotation views of the complex are shown. The fingertips are labeled. Orientation in <em>A</em> is indicated by labeling of the thumb, palm, and fingers. The linker and β-flap (see Introduction for details) are labeled
               in <em>B. C</em>, details of the H77_NS5b GTP binding site are shown. Orientation is as in <em>B</em>, with the palm domain removed for clarity. Residues contacting the putative priming GTP (labeled <em>P</em>) are shown as <em>sticks</em> and colored by element. All residues thus highlighted are strictly conserved among HCV strains, except the two <em>underlined</em>. Position 556 is always a short hydrophilic residue (Gly, Ser, Asp, or Asn), and position 446 is always an acidic residue
               that is conserved within genotypes (Glu for 1a, 2, and 5; Asp for 3, 4, and 6), except for subtype 1b, where 446 is a conserved
               Gln. <em>Pink dashes</em> denote hydrogen bonds. <em>D</em>, the same view as <em>C</em> is shown of the J4_NS5b_S556K structure. 2<em>F<sub>o</sub></em> − <em>F<sub>c</sub></em> final electron density is displayed at a 1-δ level in a 1.6 Å radius around the linker and β-flap. Note that the well defined
               lysine 556 side chain points into the region occupied by the P GTP in <em>C. E</em>, shown is the primary structure of HVC_NS5b. The C-terminal transmembrane anchor, deleted from the constructs used in this
               study, is in <em>dark gray</em>.
            </p>
            <div class="sb-div caption-clear"></div>
         </div>
      </div>

      <p id="p-9">In the present work we present a parallel structural and functional characterization of subtype 1a (H77 strain) and subtype
         1b (J4 strain) NS5b enzymes with the aim of shedding new light on the puzzle of the linker involvement in the initiation of
         RNA synthesis. Our structural results include the first crystal structures of subtype 1a H77_NS5b_Δ21 alone and in complex
         with GTP and crystal structures of subtype 1b J4_NS5b_Δ21 in a new crystal form and of a mutant of the linker. A small-angle
         x-ray scattering (SAXS) characterization of J4_NS5b_Δ21 was also performed to assess the conformation of NS5b in solution.
         Functional analyses were done by <em>in vitro</em> RdRp assays of H77_NS5b_Δ21 and J4_NS5b_Δ21 (wild type and linker mutants). Taken together, these results clarify the early
         steps of RNA synthesis by HCV-NS5b; on the one hand, they point to the direct involvement of the linker in the very first
         steps of <em>de novo</em> initiation of RNA synthesis, leading to formation of the first dinucleotide primer. On the other hand, they show that GTP
         stimulates a later transition to processive elongation that is the true rate-limiting step in initiation.
      </p>

   </div>
   <div class="section methods" id="sec-2">

      <h2>EXPERIMENTAL PROCEDURES</h2>

      <div id="sec-3" class="subsection">

         <div id="sec-4" class="subsection">

            <div id="sec-5" class="subsection">

               <h5>Site-directed Mutagenesis</h5>

               <p id="p-10">Site-directed mutagenesis of the serine 556 of H77 and J4 NS5b_Δ21 was performed by using the QuikChange site-directed mutagenesis
                  kit (Stratagene) with oligonucleotides described in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Sequences of mutated fragments were verified by DNA sequencing using the ABI Prism Big Dye terminator sequencing kit at
                  the Plateforme Genome transcriptome, Université de Bordeaux 2.
               </p>
            </div>

            <div id="sec-6" class="subsection">

               <h5>Protein Expression and Purification</h5>

               <p id="p-13">The wild type or mutant NS5b_Δ21 of H77 and J4 were expressed in <em>Escherichia coli</em> and purified as previously described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). The H77_NS5b_Δ21 used in structural studies was the Q65H mutant described in Lou <em>et al.</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>) and was purified according to the protocol therein with 0.5% <em>n</em>-octylglucoside instead of 0.1% Nonidet P-40. The wild type J4_NS5b_Δ21 used in structural studies was expressed and purified
                  according to the protocol of O'Farrell <em>et al.</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>), with minor modifications (<em>i.e.</em> 1% glucose was added to repress NS5b expression in all media, except the induction medium and carbenicillin was used as antibiotic
                  instead of ampicillin). For all preparations used in structural work, the fractions containing the purified proteins were
                  pooled, dialyzed against 5–10 m<span class="sc">m</span> Tris, pH 7.5, 0.2 <span class="sc">m</span> ammonium acetate, 1–4 m<span class="sc">m</span> DTT, 1 m<span class="sc">m</span> EDTA, flash-frozen in liquid nitrogen, and kept at −80 °C until use. Protein concentrations were determined by <em>A</em><sub>280</sub> with extinction coefficients calculated from the construct sequences.
               </p>

            </div>

            <div id="sec-7" class="subsection">

               <h5>Crystallization, Soaking, and Data Collection for H77_NS5b_Δ21</h5>

               <p id="p-14">Monoclinic crystals (H77_X1) of H77_NS5b_Δ21 were initially obtained by the hanging drop vapor diffusion method. Equal volumes
                  of protein solution (3.5 mg/ml in HEPES 8 m<span class="sc">m</span>, pH 7.5, 1.5 <span class="sc">m</span> ammonium acetate, 2.5 m<span class="sc">m</span> MgCl<sub>2</sub>, 1 m<span class="sc">m</span> EDTA, 4 m<span class="sc">m</span> DTT, also containing 55 μ<span class="sc">m</span> concentrations of a short 5-base RNA and 0.5 m<span class="sc">m</span> GTP) and well solution (0.1 <span class="sc">m</span> MES<span class="sc"></span>, pH 6.5, 0.2 <span class="sc">m</span> ammonium sulfate<span class="sc"></span>, and PEG 5000 monomethyl ether) were mixed. To improve reproducibility and quality of the crystals, streak seeding and microseeding
                  techniques were used. The final refined protocol uses the sitting drop vapor diffusion method and produces a triclinic crystal
                  form (H77_X2). Equal volumes (2 μl each) of protein solution (3.5 mg/ml in 10 m<span class="sc">m</span> HEPES<span class="sc"></span>, pH 7.0, 0.2 <span class="sc">m</span> ammonium acetate, 1 m<span class="sc">m</span> EDTA, 5 m<span class="sc">m</span> DTT) and well solution (50 m<span class="sc">m</span> MES<span class="sc"></span>, pH 6.5, 0.2 <span class="sc">m</span> ammonium sulfate, 0.25 <span class="sc">m</span> ammonium acetate, 25–30% PEG 1000) were mixed and left to equilibrate overnight. Microseeding was then carried out with a
                  dilute stock solution of crushed crystals, 0.2 μl of which was pipeted into each well. Crystals were transferred for a few
                  seconds in a cryosolution containing 20% glycerol plus all the other components of the crystallization drop and flash-cooled
                  in liquid ethane. In the case of soaks with substrates, these were added to a final concentration of 5–10 m<span class="sc">m</span> (nucleotides) and/or 0.1–1 m<span class="sc">m</span> (RNAs), MgCl<sub>2</sub> was added to 10 m<span class="sc">m</span>, no ammonium sulfate was included, and soaking time was 20–40 min. High resolution synchrotron data (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) were collected at beamline X06SA of the Swiss Light Source (Villigen, Switzerland).
               </p>
            </div>

            <div id="sec-8" class="subsection">

               <h5>Crystallization, Soaking, and Data Collection for J4_NS5b_Δ21 and J4_NS5b_S556K</h5>

               <p id="p-20">The orthorhombic crystal form (J4_O1) reported by O'Farrell <em>et al.</em> (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>) was reproduced by a different route and with significantly different cell parameters (our crystal form is hereafter called
                  J4_O2). Our protocol is very reproducible and uses the hanging drop vapor diffusion method with microseeding; seeds were initially
                  produced by crushing clusters of needles obtained by mixing equal 1-μl volumes of protein solution (6.3 mg/ml) and well solution
                  (50 m<span class="sc">m</span> ammonium acetate<span class="sc"></span>, pH 5.0, 0.5 <span class="sc">m</span> NaCl, 10% glycerol, and 15–22% PEG 4000). Equal volumes (1 μl each) of protein solution and well solution (0.2 <span class="sc">m</span> magnesium sulfate, 15–25% PEG 2000 monomethyl-ether) were then mixed and left to equilibrate overnight. Microseeding was
                  finally carried out with a 100–10,000× dilution of freshly crushed needles, 0.2 μl of which was pipeted into each drop.
               </p>

               <p id="p-21">A new, trigonal crystal form (J4_T) was obtained by lowering the salt concentration (initially 0.2 <span class="sc">m</span> ammonium acetate). Crystals were obtained by mixing 1 μl of protein solution (5 mg/ml) with 1 or 2 equal volumes of water
                  and 1 or 2 equal volumes of reservoir solution (2–6% PEG 3350, 0.2 <span class="sc">m</span> NaF). The initial dilution promoted nucleation, and the subsequent drop shrinking allowed slow (∼1 week) growth of large
                  (&gt;0.3 × 0.3 × 0.3 mm<sup>3</sup>) trigonal crystals.
               </p>

               <p id="p-22">Crystals were transferred for a few seconds in a cryosolution containing all the components of the crystallization drop plus
                  15% (J4_O2) or 30% (J4_T) glycerol and flash-cooled in liquid nitrogen. In the case of soaks with substrates (J4_O2), magnesium
                  sulfate was first substituted by 0.4 <span class="sc">m</span> ammonium acetate in 5 successive ∼2–5-fold dilutions of the drop. 10 m<span class="sc">m</span> MgCl<sub>2</sub>, 10 m<span class="sc">m</span> GTP, and/or 1 m<span class="sc">m</span> CUGGC RNA were then included, and soaking time was 30–40 min. High resolution synchrotron data (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) were collected at beamline PROXIMA1 of the SOLEIL Synchrotron (Saclay, France) or beamline ID14–1 of the European Synchrotron
                  Radiation Facility (Grenoble, France).
               </p>

            </div>

            <div id="sec-9" class="subsection">

               <h5>Structure Determination</h5>

               <p id="p-23">Data were processed with the XDS package (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). Molecular replacement was performed with program Molrep (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). Models were rebuilt with COOT (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>) and refined with phenix.refine (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). In the cases of H77_X1 and H77_X2, BK_NS5b_Δ21 was used as search model in molecular replacement (Protein Data Bank (PDB)
                  code <a href="http://www.jbc.org/external-ref?link_type=PDB&amp;access_num=1QUV">1QUV</a>), and the initial solution was first rebuilt automatically with arp_warp (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). In the cases of J4_O2 and J4_T, the previously published J4_NS5b_Δ21 (PDB <a href="http://www.jbc.org/external-ref?link_type=PDB&amp;access_num=code1NB4">code1NB4</a>) was used as the search model in molecular replacement.
               </p>

            </div>

            <div id="sec-10" class="subsection">

               <h5>Objective Comparisons of Structures</h5>

               <p id="p-24">Differences in main chain conformation between pairs of molecules were objectively assessed using program ESCET (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). We used significance values (ESCET “lolim” parameter) of 5.0 σ to identify broader conformational differences and 2.5 σ
                  to take into account all significant conformational differences.
               </p>

            </div>

            <div id="sec-11" class="subsection">

               <h5>Small Angle X-ray Scattering</h5>

               <p id="p-25">Protein samples of J4_NS5b_Δ21 were freshly dialyzed against Tris 10 m<span class="sc">m</span>, pH 7.5, ammonium acetate 200 m<span class="sc">m</span>, EDTA 1 m<span class="sc">m</span>, DTT 4 m<span class="sc">m</span> less than 48 h before and concentrated by ultrafiltration (Vivaspin, membrane cutoff, 30 kDa) to 4.0 mg/ml immediately before
                  SAXS experiments. Data were collected at beamline ID14–3 of the European Synchrotron Radiation Facility (Grenoble, France).
                  The beamline setup used was calibrated with a protein solution of known mass and concentration (bovine serum albumin at 4.5
                  mg/ml). Data were compared for radiation damage, but none was observed. Data were processed using standard procedures and
                  extrapolated to infinite dilution by the program package PRIMUS (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Program CRYSOL (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>) was used to compare this experimental SAXS pattern to those calculated from the x-ray crystal structure and from more open
                  models of J4_NS5b_Δ21. The latter models were constructed by opening the thumb and displacing the linker enough to accommodate
                  a template/primer RNA duplex (see the legend to <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>) taken from a Foot and Mouth Disease Virus RdRp complex (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>).
               </p>

            </div>

            <div id="sec-12" class="subsection">

               <h5>In Vitro RdRp Assays</h5>

               <p id="p-26">Three templates were used in these assays. The first two correspond to the 341 nt of the 3′ end of the minus strand RNA from
                  HCV genotype 1 (G1-C) or 3 (G3-U). They were obtained by <em>in vitro</em> transcription as previously described (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). The third template (G3–4U) was obtained by <em>in vitro</em> transcription of a DNA template obtained by PCR amplification from the 5′-UTR of the geno 3 clone 4 HCV with the G3–4U forward
                  primer AAAACCCTCTTACGAGGGGACACTC and the 5′-341-T7-2 reverse primer (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>).
               </p>

            </div>

            <div id="sec-13" class="subsection">

               <h5>Single-round Replication Assay</h5>

               <p id="p-27">HCV RdRp and RNA were preincubated for 30 min at 25 °C in a reaction mixture containing 20 m<span class="sc">m</span> Tris-HCl, pH 7.5, 1 m<span class="sc">m</span> DTT, 5 m<span class="sc">m</span> MgCl<sub>2</sub>, 40 m<span class="sc">m</span> NaCl, 17 units of RNasin (Promega), 0.5 m<span class="sc">m</span> concentrations of the two initiating nucleotides (GTP and CTP for RNA G1-C; ATP and CTP for RNA G3-U) or of GTP alone or
                  no NTP, 86 n<span class="sc">m</span> RNA template, and 200 n<span class="sc">m</span> purified NS5b. Heparin (<em>M</em><sub>r</sub> 4000–6000, 200 μg/ml), then [<sup>32</sup>P]UTP (3000 Ci·mmol <sup>−1</sup>, PerkinElmer Life Sciences) and the NTP needed for elongation (0.5 m<span class="sc">m</span> each) were successively added. The reaction was run at 25 °C for 0, 5, 10, 20, or 60 min and stopped by the addition of 10%
                  trichloroacetic acid (TCA). The radioactivity incorporated into neo-synthesized RNA was then determined by scintillation counting.
               </p>

            </div>

            <div id="sec-14" class="subsection">

               <h5>Gel-based Initiation and Elongation Assay</h5>

               <p id="p-28">The assay was carried out in 20 μl of 20 m<span class="sc">m</span> Tris, pH 7.5, 50 m<span class="sc">m</span> NaCl, 5 m<span class="sc">m</span> MgCl<sub>2</sub>, 1 m<span class="sc">m</span> DTT, 0.4 units/μl RNasin, 1 μ<span class="sc">m</span> RNA template, 1 μ<span class="sc">m</span> NS5b. When the initiation phase alone was analyzed, 0.5 m<span class="sc">m</span> GTP or ATP and 10 μ<span class="sc">m</span> CTP with 4 μCi of [α-<sup>32</sup>P] CTP (3000 Ci·mmol<sup>−1</sup>) were the only added nucleotides. At different time points, 4 μl were collected, the reaction was quenched by adding 1 μl
                  of 100 m<span class="sc">m</span> SDS and diluted in 8 μl of 90% formamide, 10 m<span class="sc">m</span> EDTA, 0.025% SDS, 0.025% bromphenol blue, 0.025% xylene cyanol loading buffer. After denaturation at 70 °C for 5 min, samples
                  were loaded onto a 22% polyacrylamide gel in Tris borate EDTA buffer containing 7 <span class="sc">m</span> urea. The migration was performed at 30 watts for 3 h, and the gel was submitted to electronic autoradiography using a Pharos
                  apparatus and Quantity One software (Bio-Rad). The elongation phase was analyzed after a 2-h incubation in the same assay
                  conditions except for the NTP used. GTP was added at different concentrations before or at the same time as 0.5 m<span class="sc">m</span> ATP, 3′-dUTP, and 10 or 100 μ<span class="sc">m</span> CTP with 4 μCi of [α-<sup>32</sup>P]CTP (3000 Ci·mmol<sup>−1</sup>, PerkinElmer Life Sciences). When the elongation was stopped with 0.5 m<span class="sc">m</span> 3′-dGTP instead of 3′-dUTP, UTP (0.5 m<span class="sc">m</span>) was added in the elongation reaction mixture. The reaction was stopped by adding 5 μl of 100 m<span class="sc">m</span> SDS. Five microliters were diluted in 8 μl of formamide/dyes loading buffer, and the reaction products were analyzed on polyacrylamide
                  gel as above. The molecular weight markers were synthesized as described in Dutartre <em>et al.</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>) or obtained from Dharmacon and <sup>32</sup>P-labeled with [γ-<sup>32</sup>P]ATP (3000 Ci·mmol<sup>−1</sup>, PerkinElmer Life Sciences) and T4 polynucleotide kinase following the manufacturer instructions (Promega). To enhance RNA
                  synthesis in gel-based assays with templates with a 3′-U (G3-U and G3–4U), DMSO (10%v/v) was added in the reaction mixtures
                  using these templates as well as to those using G1-C in side-by-side comparisons (<em>cf.</em> the legends to <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figs. 4</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>). Control experiments with G1-C indicated that the addition of DMSO increased 2–3-fold RNA synthesis without modifying the
                  pattern of initiation and elongation products.
               </p>

            </div>

         </div>

      </div>

   </div>
   <div class="section results" id="sec-15">

      <h2>RESULTS</h2>

      <div id="sec-16" class="subsection">

         <div id="sec-17" class="subsection">

            <div id="sec-18" class="subsection">

               <h5>Crystal Structure of H77_NS5b and Comparison with J4_NS5b</h5>

               <p id="p-29">We report here to high resolution the first structure of an NS5b of subtype 1a. The construct we used in the structural studies
                  is a Q65H point mutant of strain H77_NS5b_Δ21. The Q65H mutation was isolated from a chronically infected chimpanzee, where
                  it became fixed in the dominant virus present in the blood of the infected animal. The H77_NS5bQ65H_Δ21 construct was previously
                  found to be even more active than the wild type in <em>in vitro</em> assays (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>). We obtained two related crystal forms (X1 and X2) of H77_NS5bQ65H_Δ21 with, respectively, three and two molecules in the
                  asymmetric units, yielding five high resolution views of the subtype 1a NS5b (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). We subjected these five views to an objective conformational pairwise comparison with the program ESCET (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>). <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows that we may ascribe the five molecules to two groups; X1_A, X1_B, and X2_A on the one hand and X1_C and X2_B on the
                  other hand. The molecules are conformationally identical within each group but display small conformational differences between
                  groups. These differences are ascribable to different crystal packing environments and are significant with a tolerance of
                  2.5 σ but not of 5 σ (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, numbers in parentheses). In contrast, a structure of subtype 1b NS5b (strain J4, with nearly 90% sequence identity to H77_NS5b)
                  is significantly different from all five views of H77_NS5b even with a tolerance of 5 σ (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, <em>last column</em>). We may, therefore, break up the NS5b structure in subdomains according to the conformational differences between 1a and
                  1b enzymes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). This analysis is pictured in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>. It shows that the main conformational difference lies in the shift of a block made of the structural elements taking part
                  in the fingertips/thumb interaction, including helix A of the fingertips and helix S of the thumb. This difference leads to
                  a maximum shift in this region in C-α position of 3.5 Å between J4_ and H77_NS5b (0.6 Å overall). In contrast, the part of
                  the thumb (including the linker and β-flap) not directly involved in interactions with the fingertips is in the same rigid
                  block as the palm and fingers.
               </p>

               <div id="F2" class="fig pos-float odd">
                  <div class="fig-inline">
                    <a href="32906/F2.expansion.html" class="fig-inline-link"><img alt="FIGURE 2." src="http://www.jbc.org/content/285/43/32906/F2.small.gif"></a>
                  </div>
                  <div class="fig-caption"><span class="fig-label">FIGURE 2.</span>
                     <p id="p-32" class="first-child"><strong>Objective comparison between H77_NS5b and J4_NS5b.</strong> Shown is a ribbon view of the H77_NS5b structure in the orientation of <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a><em>A</em> and colored after ESCET comparison with J4_NS5b using a significance level of 5 σ (see the legend to <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). J4_NS5b_S556K was chosen for the comparison, as it is the best defined J4_NS5b structure available (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, 〈esd〉 line) and conformationally identical to other J4_NS5b structures at this significance level (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, <em>numbers in parentheses</em>). The conformational differences between H77_NS5b and J4_NS5b indicate two major blocks in the molecule; the first (434 residues)
                        is colored <em>blue</em>, and the second (89 residues) is in <em>green</em>, with two smaller regions (<em>cyan</em> and <em>magenta</em>); red zones indicate flexible regions. Helix A of the fingertips and helix S of the thumb are labeled.
                     </p>
                     <div class="sb-div caption-clear"></div>
                  </div>
               </div>

               <p id="p-33">Noteworthy is the absence of ligands in the H77 X1 crystal form despite the presence in the crystallization drops of both
                  a short RNA spanning the 3′-most 5 residues of the genotype 1 negative strand and GTP (see “Experimental Procedures”). Several
                  sulfate ions were visible, however, one occupying the position of the triphosphate moiety of the “priming” nucleotide previously
                  identified in our study of NS5b_1b_Δ55 in complex with nucleotides (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>).
               </p>
            </div>
</div>
